test/opf/opf.jl

```diff
@@ -36,6 +38,10 @@ const PGLIB_CASES = ["14_ieee", "30_ieee", "57_ieee", "89_pegase", "118_ieee"]
     @testset "$(OPF)" for OPF in OPFGenerator.SUPPORTED_OPF_MODELS
         @testset "$(casename)" for casename in PGLIB_CASES
             test_opf_pm(OPF, "pglib_opf_case$(casename)")
+
+            data = OPFData(network)
+            test_opfdata(data, network)
+            test_buspair_voltage_bounds(data, network)
```

Comment:
Move these tests to a separate testset. They are data-processing utilities that are independent of the OPF formulation.
It's enough to run them once (instead of once per case and per OPF formulation)

Reply:
Will do by switching the order of the loops on lines above. Also lets us only parse the file once per case (instead of per case per formulation) which will speed up the tests in general.